nbs/100_models.layers.ipynb

```diff
@@ -869,7 +869,7 @@
    "outputs": [
```

Comment:
<p>n-in is not needed in adaptive weighted pooling</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/vrodriguezf/tsai/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>